client/src/app/pages/applications/components/application-list-expanded-area/application-list-expanded-area-analysis.tsx

```diff
@@ -118,6 +118,21 @@ export const ApplicationListExpandedAreaAnalysis: React.FC<
             "Not available"
           )}
         </DescriptionListDescription>
+        {task?.state === "Succeeded" && (
+          <DescriptionListDescription>
+            <Tooltip content="Click to download Analysis report">
+              <Button variant="link" isInline>
+                <Link
+                  to={`/hub/applications/${application.id}/bucket${task?.data?.output}`}
```

Comment:
We probably want to use the defined path APPLICATIONS here.

Reply:
Good idea :+1: